src/lib/components/Table/Table.jsx

```diff
@@ -1,25 +1,18 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import { withGlobalProps } from '../../provider';
+import { transferProps } from '../_helpers/transferProps';
 import { Button } from '../..';
 import styles from './Table.scss';
 
-export class Table extends React.Component {
-  constructor(props) {
-    super(props);
-
-    this.renderHeaderCell = this.renderHeaderCell.bind(this);
-
-    this.sortCellStyle = {
-      backgroundColor: 'lightgray',
-    };
-  }
-
-  renderHeaderCell(column) {
-    const {
-      id,
-      sort,
-    } = this.props;
+export const Table = ({
+  columns,
+  id,
+  rows,
+  sort,
+  ...restProps
+}) => {
+  const renderHeaderCell = (column) => {
```

Comment:
Just personal note, because I would like to solve it in different issue. This change means that `renderHeaderCell` and `renderBodyCell` will be created and re-rendered every time when Table is re-rendered, because you are defining those components in-line. So the least change must always generate new components and render them all, React is not able to compare them between frames as they are generated each time. This differs from class methods, if I am not mistaken. This might become performance issue for larger data sets.

We can leave it here as it is and solve it later OR you can create **internal/private** components in `_components/TableHeaderCell` and `_components/TableBodyCell` to solve it now.

cc @adamkudrna

Reply:
Internal/private components were created. Can you please check if it is ok? @bedrich-schindler